SAP-BAPI-GoodsMovementCreate/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+    "title": "SAP BAPI: Create SAP GoodsMovement using BAPI_GOODSMVT_CREATE",
+    "description": "Create a goods movement in SAP using the BAPI_GOODSMVT_CREATE function module. The input is assumed to be a RFC XML document provided by a HTTP POST to the logic app. This template executes a RFC Call BAPI_GOODSMVT_CREATE on the SAP system.",
+    "prerequisites": "The [SAP .Net Connector](https://support.sap.com/en/product/connectors/msnet.html) libraries  need to be deployed under the assemblies of the Standard LogicApp, see [Connect to SAP from workflows in Azure Logic Apps](https://learn.microsoft.com/en-us/azure/logic-apps/connectors/sap?tabs=consumption) for more information. You will also need connection details for your SAP system and a RFC user.",
```

Comment:
As noted previously, this will need to contain an explanation and a link to this github template for the time being due to your extra artifacts (i.e. httpTEstFile.http, sampleXML.xml). Again, this is because we do not support the extra artfiacts at this moment, so the users will have to copy over the files themselves in order to use this template correctly. We will need a proper explanation here for this, for example, `"In order to use this template, please copy over the artifacts from [Github](https://github.com/Azure/LogicAppsTemplates).........."`

Reply:
As per the business tracking template I mentioned as a reference,
`"prerequisites": "Before you use this template, you need to meet the following prerequisites. 1) An existing 'Business Process Tracking' resource deployed Azure that emits data to a Data Explorer instance. 2) An Azure Storage account that includes a table with a field named 'LastAlertedProcessed'. 3) In the Azure portal, upload the XSLT map and LML file from the template folder to the Maps folder in your logic app resource. You can find the Maps folder under Artifacts on your logic app menu. Other artifacts in the template folder exist for your reference, for example, if you want to edit the data transformations.",`